dist/common/supervisor/scylla-jmx.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+. $(dirname "$0")/scylla_util.sh
+
+. "$sysconfdir"/scylla-jmx
+
+execsudo "$scylladir"/jmx/scylla-jmx $SCYLLA_JMX_PORT $SCYLLA_API_PORT $SCYLLA_API_ADDR $SCYLLA_JMX_ADDR $SCYLLA_JMX_FILE $SCYLLA_JMX_LOCAL $SCYLLA_JMX_REMOTE $SCYLLA_JMX_DEBUG
```

Comment:
`exec sudo`?

Reply:
and why sudo?

Reply:
> `exec sudo`?

it's
```
execsudo() {
    if is_nonroot; then
        exec "$@"
    else
        exec sudo -u scylla -g scylla "$@"
    fi
}
```
in scylla_util.sh.

Why I added it is, because scylla_prepare and scylla has to run in different user.
scylla_prepare is run in root, but scylla is run in scylla.
But it's only scylla-server problem, we don't need to do `exec sudo -u scylla` on other services such as scylla-jmx, so I will drop it and replace with `user=scylla` directive on supervisor config file.


About scylla-server, on very early version of scylla we did opposite:

scylla-server.service:
```
ExecStartPre=/usr/bin/sudo -E /usr/lib/scylla/scylla_prepare
ExecStart=/usr/bin/scylla $SCYLLA_ARGS $SEASTAR_IO $DEV_MODE
User=scylla
```

sudoers.d/scylla:
```
scylla ALL=(ALL) NOPASSWD: /usr/lib/scylla/scylla_prepare,/usr/lib/scylla/scylla_stop,/usr/lib/scylla/scylla_io_setup,/usr/lib/scylla/scylla-ami/scylla_ami_setup
```

But it wasn't enough, we also needed to rewrite sudoers:
```
%if 0%{?rhel}
sed -e "s/Defaults    requiretty/#Defaults    requiretty/" /etc/sudoers > /tmp/sudoers
cp /tmp/sudoers /etc/sudoers
rm /tmp/sudoers
%endif
```

It seems like to run `sudo -u scylla` in root is much easier than opposite way, since there are restrictions to become root and it should be doing carefully.

Reply:
Ah, ok. But sudo suggests that it will run as root, suggest calling it run_as_scylla_user to be super clear.